src/components/MyPortfolio/myportfolio.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { MyPortfolio } from '.';
+
+export default {
+  component: MyPortfolio,
+  title: 'Components/MyPortfolio',
+};
+
+const TemplateMyPortfolio: ComponentStory<typeof MyPortfolio> = (args: any) => (
+  <div style={{ width: 600 }}>
+    <MyPortfolio {...args} />
```

Comment:
1. in storybook selected item is dark and not readable. 
2. portfolio total value is $1,519.13 but in sidebar if you see, it shows really high values. 

you can try this address `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266`

![Screenshot 2022-06-16 at 4 03 44 PM](https://user-images.githubusercontent.com/6604146/174052129-6ce72180-0dac-4e98-8178-ec536c41ee5c.png)